components/event-publisher-proxy/pkg/sender/nats.go

```diff
@@ -19,14 +20,14 @@ type GenericSender interface {
 
 // HttpMessageSender is responsible for sending messages over HTTP.
 type NatsMessageSender struct {
-	Ctx    context.Context
-	Target string
-	Logger *logrus.Logger
+	Ctx        context.Context
```

Comment:
Do `Ctx` and `Logger` need to be public?

Reply:
Done.